From: Qingqing <wangyiqing@kylinos.cn>
Date: Tue, 21 Jul 2026 09:35:23 +0000
Subject: Translated using Weblate (Lao)

Currently translated at 100.0% (28 of 28 strings)

Translation: openkylin-huanghe/mobilehotspot
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/mobilehotspot/lo/
---
 plugins/mobilehotspot/translations/lo.ts | 56 ++++++++++++++++----------------
 1 file changed, 28 insertions(+), 28 deletions(-)

diff --git a/plugins/mobilehotspot/translations/lo.ts b/plugins/mobilehotspot/translations/lo.ts
index f2d8b02..1c0338b 100644
--- a/plugins/mobilehotspot/translations/lo.ts
+++ b/plugins/mobilehotspot/translations/lo.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../blacklistitem.cpp" line="50"/>
         <source>Remove</source>
-        <translation type="unfinished"></translation>
+        <translation>ຍ້າຍອອກ</translation>
     </message>
 </context>
 <context>
@@ -14,7 +14,7 @@
     <message>
         <location filename="../blacklistpage.cpp" line="48"/>
         <source>Blacklist</source>
-        <translation type="unfinished"></translation>
+        <translation>ບັນຊີດຳຂອງອຸປະກອນ</translation>
     </message>
 </context>
 <context>
@@ -22,7 +22,7 @@
     <message>
         <location filename="../connectdevlistitem.cpp" line="50"/>
         <source>drag into blacklist</source>
-        <translation type="unfinished"></translation>
+        <translation>ເພີ່ມລາຍຊື່ເຂົ້າໃນບັນຊີດຳ</translation>
     </message>
 </context>
 <context>
@@ -30,7 +30,7 @@
     <message>
         <location filename="../connectdevpage.cpp" line="49"/>
         <source>Connect device</source>
-        <translation type="unfinished"></translation>
+        <translation>ເຄືອຂ່າຍທີ່ເຊື່ອມຕໍ່</translation>
     </message>
 </context>
 <context>
@@ -38,54 +38,54 @@
     <message>
         <location filename="../mobilehotspot.cpp" line="35"/>
         <source>MobileHotspot</source>
-        <translation type="unfinished"></translation>
+        <translation>ຮ໋ອດສະປັອດມືຖື</translation>
     </message>
     <message>
         <location filename="../mobilehotspot.cpp" line="110"/>
         <source>mobilehotspot open</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດຮ໋ອດສະປັອດ</translation>
         <extra-contents_path>/mobilehotspot/mobilehotspot open</extra-contents_path>
     </message>
     <message>
         <location filename="../mobilehotspot.cpp" line="112"/>
         <source>Mobile Hotspot</source>
-        <translation type="unfinished"></translation>
+        <translation>ຮ໋ອດສະປັອດມືຖື</translation>
         <extra-contents_path>/mobilehotspot/Mobile Hotspot</extra-contents_path>
     </message>
     <message>
         <location filename="../mobilehotspot.cpp" line="114"/>
         <source>Open mobile hotspot</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດຮ໋ອດສະປັອດມືຖື</translation>
         <extra-contents_path>/mobilehotspot/Open mobile hotspot</extra-contents_path>
     </message>
     <message>
         <location filename="../mobilehotspot.cpp" line="116"/>
         <source>Wi-Fi Name</source>
-        <translation type="unfinished"></translation>
+        <translation>ຊື່ Wi-Fi</translation>
         <extra-contents_path>/mobilehotspot/Wi-Fi Name</extra-contents_path>
     </message>
     <message>
         <location filename="../mobilehotspot.cpp" line="118"/>
         <source>Network Password</source>
-        <translation type="unfinished"></translation>
+        <translation>ລະຫັດຜ່ານເຄືອຂ່າຍ</translation>
         <extra-contents_path>/mobilehotspot/Network Password</extra-contents_path>
     </message>
     <message>
         <location filename="../mobilehotspot.cpp" line="120"/>
         <source>Network Frequency band</source>
-        <translation type="unfinished"></translation>
+        <translation>ຊ່ວງຄວາມຖີ່ເຄືອຂ່າຍ</translation>
         <extra-contents_path>/mobilehotspot/Network Frequency band</extra-contents_path>
     </message>
     <message>
         <location filename="../mobilehotspot.cpp" line="122"/>
         <source>Shared NIC port</source>
-        <translation type="unfinished"></translation>
+        <translation>ແບ່ງປັນພອດແຄດ</translation>
         <extra-contents_path>/mobilehotspot/Shared NIC port</extra-contents_path>
     </message>
     <message>
         <location filename="../mobilehotspot.cpp" line="108"/>
         <source>mobilehotspot</source>
-        <translation type="unfinished"></translation>
+        <translation>ຮ໋ອດສະປັອດມືຖື</translation>
         <extra-contents_path>/mobilehotspot/mobilehotspot</extra-contents_path>
     </message>
 </context>
@@ -102,7 +102,7 @@
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="165"/>
         <source>wirless switch is close or no wireless device</source>
-        <translation type="unfinished"></translation>
+        <translation>ປຸ່ມເປີດ/ປິດ WiFi ຖືກປິດ ຫຼື ບໍ່ມີແຄດ WiFi ທີ່ສະຫນັບສະຫນູນຮ໋ອດສະປັອດ</translation>
     </message>
     <message>
         <source>start to close hotspot</source>
@@ -111,7 +111,7 @@
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="190"/>
         <source>hotpots name or device is invalid</source>
-        <translation type="unfinished"></translation>
+        <translation>ຊື່ຮ໋ອດສະປັອດ ຫຼື ອຸປະກອນຜິດພາດ</translation>
     </message>
     <message>
         <source>can not  create hotspot with password length less than eight!</source>
@@ -124,32 +124,32 @@
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="644"/>
         <source>Contains at least 8 characters</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕ້ອງມີຢ່າງໜ້ອຍ 8 ຕົວອັກສອນ</translation>
     </message>
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="245"/>
         <source>Hotspot</source>
-        <translation type="unfinished"></translation>
+        <translation>ຮ໋ອດສະປັອດມືຖື</translation>
     </message>
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="64"/>
         <source>Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕັ້ງຄ່າ</translation>
     </message>
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="67"/>
         <source>Settings desktop message</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕັ້ງຄ່າການແຈ້ງເຕືອນແບບ Desktop</translation>
     </message>
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="1103"/>
         <source>use </source>
-        <translation type="unfinished"></translation>
+        <translation>ໃຊ້ </translation>
     </message>
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="1104"/>
         <source> share network, will interrupt local wireless connection</source>
-        <translation type="unfinished"></translation>
+        <translation> ການແບ່ງປັນຮ໋ອດສະປັອດຈະເຮັດໃຫ້ການເຊື່ອມຕໍ່ WiFi ຂອງອຸປະກອນນີ້ຕັດການເຊື່ອມຕໍ່</translation>
     </message>
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="381"/>
@@ -158,27 +158,27 @@
         <location filename="../mobilehotspotwidget.cpp" line="526"/>
         <location filename="../mobilehotspotwidget.cpp" line="791"/>
         <source>hotspot already close</source>
-        <translation type="unfinished"></translation>
+        <translation>ຮ໋ອດສະປັອດຖືກປິດໃຊ້ງານແລ້ວ</translation>
     </message>
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="550"/>
         <source>Open mobile hotspot</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດຮ໋ອດສະປັອດມືຖື</translation>
     </message>
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="633"/>
         <source>Network Password</source>
-        <translation type="unfinished"></translation>
+        <translation>ລະຫັດຜ່ານເຄືອຂ່າຍ</translation>
     </message>
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="672"/>
         <source>Network Frequency band</source>
-        <translation type="unfinished"></translation>
+        <translation>ຊ່ວງຄວາມຖີ່ເຄືອຂ່າຍ</translation>
     </message>
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="697"/>
         <source>Shared NIC port</source>
-        <translation type="unfinished"></translation>
+        <translation>ແບ່ງປັນພອດແຄດ</translation>
     </message>
     <message>
         <source>Open</source>
@@ -187,7 +187,7 @@
     <message>
         <location filename="../mobilehotspotwidget.cpp" line="611"/>
         <source>Wi-Fi Name</source>
-        <translation type="unfinished"></translation>
+        <translation>ຊື່ Wi-Fi</translation>
     </message>
     <message>
         <source>Password</source>
@@ -201,7 +201,7 @@
         <location filename="../mobilehotspotwidget.cpp" line="823"/>
         <location filename="../mobilehotspotwidget.cpp" line="835"/>
         <source>hotspot already open</source>
-        <translation type="unfinished"></translation>
+        <translation>ຮ໋ອດສະປັອດຖືກເປີດໃຊ້ງານແລ້ວ</translation>
     </message>
 </context>
 </TS>
